service/src/main/java/uk/gov/hmcts/reform/fpl/service/sdo/StandardDirectionsOrderService.java

```diff
@@ -44,7 +45,8 @@ public StandardDirectionOrder buildTemporarySDO(CaseData caseData) {
             // been through once, either pull from replacement doc or SDO if that isn't present
             document = defaultIfNull(
                 caseData.getReplacementSDO(),
-                caseData.getStandardDirectionOrder().getOrderDoc()
+                isNotEmpty(caseData.getStandardDirectionOrder())
```

Comment:
I am worried about what will be happening in the scenario that a user goes through the event once uploading a document then goes through again but doesn't upload a replacement document. From my understanding of the bug this will just mean that the new SDO is populated with a null document instead of the previous one